packages/opentelemetry-api/README.md

```diff
@@ -9,12 +9,112 @@ This package provides everything needed to interact with the OpenTelemetry API,
 
 ## Basic Use
 
-### API Entry Point
+To get started tracing you need to install the SDK and plugins, create a TracerProvider, and register it with the API.
+
+### Install Dependencies
+
+```sh
+$ npm install \
+    @opentelemetry/core \
+    @opentelemetry/node \
+    @opentelemetry/tracing \
+    @opentelemetry/exporter-jaeger \
+    @opentelemetry/exporter-prometheus \ # add exporters as needed
```

Comment:
we should add `@opentelemetry/metrics`, if showing both Trace and Metrics example

Reply:
Or I would prefer to see separate installation for Trace and Metric quick start.